<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Aggregator</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="resources.qrc" >
    <normaloff>:/resources/images/aggregator.png</normaloff>:/resources/images/aggregator.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>34</y>
     <width>800</width>
     <height>545</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="QSplitter" name="MainSplitter_" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Tags:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TagsLineEdit" name="TagsLine_" />
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="Feeds_" >
          <property name="rootIsDecorated" >
           <bool>false</bool>
          </property>
          <property name="itemsExpandable" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ItemInfo_" >
          <property name="title" >
           <string>Item info</string>
          </property>
          <layout class="QFormLayout" name="formLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="label_4" >
             <property name="text" >
              <string>Link:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLabel" name="ItemLink_" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_13" >
             <property name="text" >
              <string>Author:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QLabel" name="ItemAuthor_" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label_15" >
             <property name="text" >
              <string>Category:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QLabel" name="ItemCategory_" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QLabel" name="label_17" >
             <property name="text" >
              <string>Publication date:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" >
            <widget class="QDateTimeEdit" name="ItemPubDate_" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
             <property name="calendarPopup" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ChannelDashboard_" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>140</height>
           </size>
          </property>
          <property name="title" >
           <string>Channel dashboard</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2" >
           <property name="fieldGrowthPolicy" >
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_5" >
             <property name="text" >
              <string>Tags:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="TagsLineEdit" name="ChannelTags_" />
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label_10" >
             <property name="text" >
              <string>Link:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QLabel" name="ChannelLink_" >
             <property name="openExternalLinks" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QLabel" name="label_8" >
             <property name="text" >
              <string>Description:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" >
            <widget class="QLabel" name="ChannelDescription_" >
             <property name="text" >
              <string/>
             </property>
             <property name="scaledContents" >
              <bool>true</bool>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0" >
            <widget class="QLabel" name="label_6" >
             <property name="text" >
              <string>Author:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" >
            <widget class="QLabel" name="ChannelAuthor_" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="0" >
            <widget class="QLabel" name="label_7" >
             <property name="text" >
              <string>Language:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" >
            <widget class="QLabel" name="ChannelLanguage_" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLabel" name="ChannelImage_" >
             <property name="text" >
              <string>&lt;channel image></string>
             </property>
            </widget>
           </item>
          </layout>
          <zorder>label_5</zorder>
          <zorder>ChannelTags_</zorder>
          <zorder>label_8</zorder>
          <zorder>ChannelDescription_</zorder>
          <zorder>label_6</zorder>
          <zorder>ChannelAuthor_</zorder>
          <zorder>label_7</zorder>
          <zorder>ChannelLanguage_</zorder>
          <zorder>ChannelLink_</zorder>
          <zorder>label_10</zorder>
          <zorder>ChannelImage_</zorder>
         </widget>
        </item>
       </layout>
       <zorder>Feeds_</zorder>
       <zorder>ChannelDashboard_</zorder>
       <zorder>ItemInfo_</zorder>
      </widget>
      <widget class="QWidget" name="layoutWidget" >
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLineEdit" name="SearchLine_" />
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="SearchType_" >
            <item>
             <property name="text" >
              <string>Fixed string</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Wildcard</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Regexp</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QCheckBox" name="CaseSensitiveSearch_" >
            <property name="text" >
             <string>Case sensitive</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTreeView" name="Items_" >
           <property name="selectionMode" >
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="rootIsDecorated" >
            <bool>false</bool>
           </property>
           <property name="itemsExpandable" >
            <bool>false</bool>
           </property>
          </widget>
          <widget class="BrowserWidget" native="1" name="ItemView_" />
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>579</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>138</width>
     <height>34</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="ActionAddFeed_" />
   <addaction name="ActionRemoveFeed_" />
   <addaction name="ActionUpdateSelectedFeed_" />
   <addaction name="ActionUpdateFeeds_" />
  </widget>
  <widget class="QToolBar" name="toolBar_2" >
   <property name="geometry" >
    <rect>
     <x>138</x>
     <y>0</y>
     <width>185</width>
     <height>34</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="ActionItemBucket_" />
   <addaction name="ActionRegexpMatcher_" />
   <addaction name="separator" />
   <addaction name="ActionPreferences_" />
  </widget>
  <widget class="QToolBar" name="toolBar_3" >
   <property name="geometry" >
    <rect>
     <x>323</x>
     <y>0</y>
     <width>477</width>
     <height>34</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar_3</string>
   </property>
   <property name="iconSize" >
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="ActionHideReadItems_" />
  </widget>
  <action name="ActionAddFeed_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >
     <normaloff>:/resources/images/rss-feed.png</normaloff>:/resources/images/rss-feed.png</iconset>
   </property>
   <property name="text" >
    <string>Add feed...</string>
   </property>
  </action>
  <action name="ActionPreferences_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >
     <normaloff>:/resources/images/preferences.png</normaloff>:/resources/images/preferences.png</iconset>
   </property>
   <property name="text" >
    <string>Preferences</string>
   </property>
  </action>
  <action name="ActionUpdateFeeds_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >
     <normaloff>:/resources/images/updateallfeeds.png</normaloff>:/resources/images/updateallfeeds.png</iconset>
   </property>
   <property name="text" >
    <string>Update feeds</string>
   </property>
  </action>
  <action name="ActionRemoveFeed_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >
     <normaloff>:/resources/images/removefeed.png</normaloff>:/resources/images/removefeed.png</iconset>
   </property>
   <property name="text" >
    <string>Remove feed</string>
   </property>
  </action>
  <action name="ActionMarkItemAsUnread_" >
   <property name="text" >
    <string>Mark item as unread</string>
   </property>
  </action>
  <action name="ActionMarkChannelAsRead_" >
   <property name="text" >
    <string>Mark channel as read</string>
   </property>
  </action>
  <action name="ActionMarkChannelAsUnread_" >
   <property name="text" >
    <string>Mark channel as unread</string>
   </property>
  </action>
  <action name="ActionUpdateSelectedFeed_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >
     <normaloff>:/resources/images/updateselectedfeed.png</normaloff>:/resources/images/updateselectedfeed.png</iconset>
   </property>
   <property name="text" >
    <string>Update selected feed</string>
   </property>
  </action>
  <action name="ActionAddToItemBucket_" >
   <property name="text" >
    <string>Add to item bucket</string>
   </property>
  </action>
  <action name="ActionItemBucket_" >
   <property name="icon" >
    <iconset resource="resources.qrc" >
     <normaloff>:/resources/images/favorites.png</normaloff>:/resources/images/favorites.png</iconset>
   </property>
   <property name="text" >
    <string>Item bucket...</string>
   </property>
  </action>
  <action name="ActionRegexpMatcher_" >
   <property name="text" >
    <string>Regexp matcher...</string>
   </property>
  </action>
  <action name="ActionHideReadItems_" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources.qrc" >
     <normaloff>:/resources/images/rss-show.png</normaloff>
     <normalon>:/resources/images/rss-hide.png</normalon>:/resources/images/rss-show.png</iconset>
   </property>
   <property name="text" >
    <string>Show/hide read items</string>
   </property>
   <property name="toolTip" >
    <string>Show/hide read items</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BrowserWidget</class>
   <extends>QWidget</extends>
   <header>browserwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TagsLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../../plugininterface/tagslineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc" />
 </resources>
 <connections/>
</ui>
